admin/tim_limitless_consts.php

```diff
@@ -8,18 +8,22 @@
 const TIM_LIMITLESS_POST_HASH_CONTENT = 'tim_limitless_post_hash_content';
 const TIM_LIMITLESS_POST_HASH_CONTENT_EXCERPT = 'tim_limitless_post_hash_content_excerpt';
 const TIM_LIMITLESS_POST_HASH_CONTENT_EXCERPT_TITLE = 'tim_limitless_post_hash_content_excerpt_title';
```

Comment:
@borisjaskerovich since these consts are being used in DB entries - I didn't want to change them, but the names should be changed too

Reply:
you can change them. we just need to do bulk update after the change.